src/clj/rems/api/resources.clj

```diff
@@ -47,10 +47,12 @@
 
     (GET "/:resource-id" []
       :summary "Get resource by id"
-      :roles #{:owner :handler}
+      :roles #{:owner :organization-owner :handler}
       :path-params [resource-id :- (describe s/Int "resource id")]
       :return Resource
-      (ok (resource/get-resource resource-id)))
+      (if-let [resource (resource/get-resource resource-id)]
+        (ok resource)
+        (not-found-json-response)))
```

Comment:
Maybe this could be named `not-found-response` since the API does not return JSON only?

Reply:
Forget this one. The actual code is a hack that should be improved elsewhere.

Reply:
Ok.